dev/AccessControl/GetSecurityDescriptorForAppContainerNames.cpp

```diff
@@ -0,0 +1,100 @@
+﻿#include <pch.h>
+#include <wil/resource.h>
+#include <wil/result.h>
```

Comment:
CONSIDER: moving WIL after the OS headers

General rule of thumb, you want to include headers from most generic to most specific e.g.

```
#include <windows.h>
#include <windows headers like aclapi.h>
#include <C/C++ language headers like vector>
#include <other libraries like WIL, WRL, etc>
#include "feature headers like Security.AccessControl.h"
```